package.json

```diff
@@ -48,7 +48,7 @@
     "eslint:ci": "eslint . --report-unused-disable-directives --ext .js,.ts,.tsx --max-warnings 0",
     "stylelint": "stylelint --reportInvalidScopeDisables --reportNeedlessDisables \"docs/**/*.?(c|m)[jt]s?(x)\" --ignore-path .gitignore",
     "markdownlint": "markdownlint-cli2 \"**/*.md\"",
-    "valelint": "git ls-files | grep -h \".md$\" | xargs vale --filter='.Level==\"error\"'",
+    "valelint": "git ls-files | grep -h \".md$\" | xargs pnpm vale --filter='.Level==\"error\"'",
```

Comment:
Does that work? `vale` is not a node comande. It's script to install on your computer https://vale.sh/docs/install#package-managers